pkg/workspace/controllers/workspace_controller.go

```diff
@@ -763,6 +768,8 @@ func (c *WorkspaceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(&karpenterv1.NodeClaim{}, c.watchNodeClaims(), builder.WithPredicates(nodeclaim.NodeClaimPredicate)).
 		WithOptions(controller.Options{MaxConcurrentReconciles: 5})
 
+	go monitorWorkspaces(context.Background(), c.Client)
```

Comment:
```suggestion
	go monitorWorkspaces(mgr.GetContext(), c.Client)
```

Reply:
I see you want to get context from the mgr, but GetContext does not exit in the mgr package? @ishaansehgal99